.github/pull_request_template.md

```diff
@@ -1,31 +1,22 @@
-## Change description
+## Jira Task
 
-> Description here
+Task: [ZDI-X](https://zeplin.atlassian.net/browse/ZDI-X)
```

Comment:
Since this is a public repo, it would be nice to keep everything reachable by everybody. So, let's remove the Jira task related things from this PR.

Reply:
✅